guides/tutorials/codelabs/azure-vmss-source-to-prod/index.md

```diff
@@ -0,0 +1,326 @@
+  * Cluster: codelab-test-tutorial (or the name of your cluster if you have used different values)
+  * Target: Newest Server Group
+
+    ![Enable test Deployment](images/3-enable-test-deployment.png)
+
+### VII. Manual Judgement 
+
+1. Click *Add stage* in the *Deploy codelab* section.
+1. Use the following values: 
+  * Type: Manual Judgment
+  * Name: Manual Judgment
+  * Depends On: Enable Test Deployment
+  * Instructions: Validate test cluster 
+
+Save the pipeline by clicking the *Save Changes* button at the bottom right.
+
+## 4. Promote to Prod
+
+In this section we will add the steps that will promote the deployment to test in production.
+
+### I. Add Deploy to prod stage
+
+1. Click *Add stage* in the *Promote to Prod* section. 
+1. Use the following values to configure the stage:
+  * Type: Deploy
+  * Stage Name: Deploy to Prod
+  * Depends On: Manual Judgment
+
+### II. Create Production deployment configuration 
+
+1. Click *Add server group* in the Deploy Configuration section. 
+1. Click *Continue without a template*
+1. In the *Configure Deployment Cluster* enter the following values:
+  * Account: my-azure-account
+  * Region: westus (or select the region that you have selected when creating the Security Group)
+  * Stack: prod
+  * Details: tutorial
+  * Load Balancers: codelab-prod-frontend
+
+    ![Configure Prod Cluster - 1](images/4-configure-prod-cluster-1.png)
+
+  * Subnets: devopsSubnet2
+  * Security Groups: codelab-web
+
+    ![Configure Prod Cluster - 2](images/3-configure-deployment-cluster-2.png)
+
+  * "Advanced Settings": 
+    - Custom Data: PROD
+    - Custom Script: https://raw.githubusercontent.com/azure-devops/hello-karyon-rxnetty/master/scripts/setcustomenv.sh
+    - Command To Execute: sudo bash ./setcustomenv.sh
+  
+      ![Configure Deployment Cluster - 3](images/4-configure-prod-cluster-2.png)
+
+  * Click **Add** 
+
+> NOTE: Record the name of the cluster. it will be used in the next step.
+
+### III. Enable Prod deployment
+
+1. Click *Add stage* in the *Deploy codelab* section.
+1. Use the following values to configuret this stage: 
+  * Type: Enable Server Group
+  * Stage Name: Enable Prod Deployment
+  * Depends On: Deploy to Prod
+  * Account: my-azure-account
+  * Regions: westus (or select the region that you have selected when creating the Security Group)
+  * Cluster: codelab-prod-tutorial
+  * Target: Newest Server Group
+
+    ![Enable Prod deployment](images/4-enable-prod-deployment.png)
+
+  * Click **Save Changes**
+
+## 5. Test it out 
+
+In this step you will try out the pipeline that you have just build. 
+
+1. Got to [Jenkins](http://localhost:8080) and launch a new build of the "Sample Aptly Job" like you did in the [*Configure Jenkins* section](#1-configure-jenkins). After some minutes, you should see your pipeline running. 
+
+   You can also click on *Start Manual Execution* on your pipeline. 

```

Comment:
Remove this line - it's confusing which pipeline you're talking about and I think we want people to exercise the whole flow